src/components/navigation-header/components/dropdown-menu/index.tsx

```diff
@@ -41,7 +41,7 @@ import s from './dropdown-menu.module.css'
  * The icons supported in this menu in addition to the Product logo icons.
  */
 const supportedIcons: { [key in SupportedIcon]: ReactElement } = {
-  box: <IconBox16 />,
```

Comment:
Seemed like 'box' wasn't used anywhere else. Removed it to minimize size for this map, lmk if that's an issue.